src/components/post-card/PostCard.js

```diff
@@ -107,15 +109,18 @@ class PostCard extends PureComponent {
             </footer>
           </blockquote>
         </div>
-
-        {
+        {/*Показуєм коментарі тільки на сторінці поста коли роут = "/posts/:id" */}
+        { !!withCommentsLoading &&
           <label onClick={this.onToggleComments} className="btn btn-link">{showComments ? 'Hide comments' : 'Show comments'}</label>
         }
         { !!error &&  <div>{error}</div> }
 
-        {/* todo 3 : добавить ссылку Link на урлу с айди поста, где будут детали поста
+        {/*  3 : добавить ссылку Link на урлу с айди поста, где будут детали поста
                     по аналогии с 24 строкой в UserCard
         */}
+        {/**/}
+
+        { (url === "/posts") && <Link to={`${url}/${post.id}`}>Show details</Link> }
```

Comment:
Or you can check if `id` from `match` exists
in case if no id - there should be link